### PR TITLE
Ahmad/Seperate Y Axis

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**
This pull request removes the Y-axis from the specified chart visualization to enhance clarity and focus on the X-axis data representation. This change is in line with the recent design updates aimed at simplifying our visualizations.  We've implemented a technique where the canvas is clipped for all drawing painters, ensuring that any drawings beyond the bounds of the chart are neatly trimmed.

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by farhan-nurzi-deriv — APPROVED** [review]

File: src/components/FastMarker.tsx [hunk 1]
```diff
@@ -156,8 +156,7 @@ const FastMarker = (props: TFastMarkerProps) => {
 
     const { overlap_y_axis = true } = props;
 
-    const maxWidth = chartNode && !overlap_y_axis ? chartNode.offsetWidth - yAxisWidth - 10 : '100%';
-
+    const maxWidth = chartNode && !overlap_y_axis ? chartNode.offsetWidth - yAxisWidth - 50 : 'calc(100% - 50px)';
```
Comment (balakrishna-deriv) [inline src/components/FastMarker.tsx:159]: is 50px the width of y-axis?

Comment (ahmadtaimoor-deriv) [reply src/components/FastMarker.tsx:159]: yes, Its the width of y-axis , we getting yAxisWidth value from the chartStore as well but it is different (60px) and setting it allowing marker to hide before it is overlapped with y-axis

**Review by kate-deriv — APPROVED** [review]

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**